lib/fetching.js

```diff
@@ -1,4 +1,5 @@
 // Sets up the fetching module process and necessary event proxies.
+'use strict';
```

Comment:
Are we going to move toward doing this project-wide? Should probably have a style document somewhere...